compiler/circle-execution-plan/src/ExecutionPlanner.cpp

```diff
@@ -40,6 +40,37 @@ uint32_t compute_output_size(luci::Padding padding, uint32_t image_size, uint32_
   }
 }
 
+bool isExecutableNode(const luci::CircleNode *node)
+{
+  switch (node->opcode())
+  {
+    // The following nodes denote outputs of multiple-output nodes.
+    case luci::CircleOpcode::CIRCLEIFOUT:
+    case luci::CircleOpcode::CIRCLESPLITOUT:
+    case luci::CircleOpcode::CIRCLESPLITVOUT:
+    case luci::CircleOpcode::CIRCLEUNPACKOUT:
+    case luci::CircleOpcode::CIRCLEWHILEOUT:
+      return false;
+    default:
+      return true;
+  }
+}
+
+bool isTensorProducingNode(const luci::CircleNode *node)
+{
+  switch (node->opcode())
+  {
+    // The following nodes are multiple-output nodes. They do not produce tensors, the tensors
+    // are produced by the corresponding *Out nodes instead.
+    case luci::CircleOpcode::IF:
+    case luci::CircleOpcode::SPLIT:
+    case luci::CircleOpcode::UNPACK:
```

Comment:
Why `While` and `SplitV` are not in the list?

Reply:
@binarman,
 This list of nodes is from the list of `luci-interpreter/src/loader/GraphLoader.cpp` of method `isTensorProducingNode`. And I would like to maintain the same list of operations.
@seanshpark,
The same reason I include only this operations 
```cxx 
    // The following nodes denote outputs of multiple-output nodes.
    case luci::CircleOpcode::CIRCLEIFOUT:
    case luci::CircleOpcode::CIRCLESPLITOUT:
    case luci::CircleOpcode::CIRCLESPLITVOUT:
    case luci::CircleOpcode::CIRCLEUNPACKOUT:
    case luci::CircleOpcode::CIRCLEWHILEOUT:
```
This list of operations is copy from `luci-interpreter/src/loader/GraphLoader.cpp` of method `isExecutableNode`

Is it better to add more operations to both lists?

Reply:
I think it is OK to sync with `GraphLoader.cpp` with a NOTE that these items are sync with `GraphLoader.cpp` :)

Reply:
I agree about sync.

I think we should add `while` and `splitV` operations in list of non producing tensor operations, because they really do not produce tensors. I'll try to test this part of interpreter more.